pfcpiface/node.go

```diff
@@ -154,3 +156,20 @@ func (node *PFCPNode) Done() {
 	<-node.done
 	log.Infoln("Shutdown complete")
 }
+
+func (node *PFCPNode) findPFCPConnById(fseid uint64) *PFCPConn {
+	connID := uint32(fseid >> 32)
+
+	for _, pConn := range node.pConns {
+		if pConn.pConnID == connID {
+			return pConn
+		}
+		continue
+	}
+	return nil
+}
+
+func (node *PFCPNode) getConnId() uint32 {
+	node.connId++
```

Comment:
How the `connId` is released if a PFCP connection goes down?

Reply:
Working on suggested changes and loop holes.